service/dap/server.go

```diff
@@ -743,7 +736,7 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 		return
 	}
 
-	if mode == "debug" || mode == "test" {
+	if mode := args.Mode; mode == DebugLaunchMode || mode == TestLaunchMode {
```

Comment:
I know in general constants are considered a good practice because they code more readable, easier to change by only updating one place, etc. But in this case, I actually think they are not helpful :(

Reply:
Agree. :-(
Changed to "debug" and "test', const literals.
Still have LaunchMode and AttachMode consts in types.go - not sure if they are good for documentation purpose.
What do you think? Should I remove them too and go back to 'string'?

Reply:
Saw the other comment. Yes, it's my misunderstanding. Removed the types and reverted back to plain string.